src/cuda/check_fields_cuda.cu

```diff
@@ -85,6 +85,21 @@ void pause_for_debugger(int pause) {
   }
 }
 
+/* ---------------------------------------------------------------------------------------- */
+
+void cudaMemoryTest(int posId)
```

Comment:
just a remark about the naming conventions: 
`cudaMemoryTest` is pretty C/C++-code style. for SPECFEM, we would usually adopt the fortran coding style `cuda_Memory_Test` or `cuda_memory_test` even for these C/CUDA-code routines.

Reply:
Noted, thank you!